### ISSUE TITLE
Login part!

### ISSUE THREAD

**Issue by andyandy22** [ztizzlegaming/CSGOWinBig#34]
I did all as you said and I have a big issue!!
www.ned-tech.com after you login it doesn't do anything. please help!!

**Comment by RatexIndex** [comment]
Your login is not working - Please Read all with the best consentration (README).

You write this link as www.ned-tech.com but after click on the login button it says  ned-tech.com so you need to add in steam auth and in all html the same better is this on all files -> http://ned-tech.com

**Comment by andrewda** [comment]
Yea, I assume you didn't set up the SteamAuth files correctly. Take a look in the SteamAuth/settings.php folder and see make sure everything is set.

**Comment by andrewda** [comment]
https://github.com/ztizzlegaming/CSGOWinBig/tree/master/src/php/SteamAuthentication/steamauth/settings.php

**Comment by andrewda** [comment]
And make sure your `passwords.txt` file is set up.

**Comment by RatexIndex** [comment]
The Password.txt makes no sense on this login error about the page.
@andrewda his fail is only the login on the page not the bot login ;)

**Comment by andrewda** [comment]
@RatexIndex not even if his API key isn't correct? If not, my bad.

**Comment by RatexIndex** [comment]
@andrewda no on this problem go to the his site and login to this page and you see www is switching to http:// ;) and this is not correnctly. He must change all time the lines with www or http://

**Comment by andrewda** [comment]
@RatexIndex forgive me if this is a trivial question, but isn't the API key set in a passwords.txt file on the website? There is a reference to it in the SteamAuthentication/steamauth/settings.php file: `$fileLoc = $_SERVER['DOCUMENT_ROOT'] . '/../passwords.txt';`

**Comment by RatexIndex** [comment]
yes thats right @andrewda but the problem is a other i think ;)
And we need a resonse first before we can say more :dash:

**Comment by andrewda** [comment]
@RatexIndex ok, just making sure I wasn't completely missing something!

**Comment by sucknorris** [comment]
Hey there,
have the same issue. After click on Login button i go forward to Steam-Login that i type in my Username and Password. After click on login i come on a site where is just a message:

"no file found"

i found that Text in the settings.php
But what is missing to fix that one -.-

**Comment by andrewda** [comment]
@sucknorris I believe the it cannot find your `passwords.txt` file. Verify that it exists.

**Comment by sucknorris** [comment]
Yeah i checked the "passwords.txt" its in the same Format like shown on this site:
"{"default-password":"YOUR DATABASE PASSWORD","steamAPIKey":"YOUR STEAM API KEY"}"
and i was so desperate that i put this sick file in each folder :-)

**Comment by RatexIndex** [comment]
@andrewda sorry atm i have no brain for issues ^^
I make updates on my fork

**Comment by andrewda** [comment]
@sucknorris I believe it goes in the root folder, in the folder containing `public_html`. This is for safety reasons as we don't want just anyone being able to access your `passwords.txt` file. If you don't have access to the root for whatever reason, you can safely update your `settings.php` to look like [this](https://gist.github.com/andrewda/ace4ed0ce7b5ef5b4429), and replace `XXX` with your API key.

**Comment by andyandy22** [comment]
Thank you all for your fast responds!! 

I understood that I need to change to be one single link like www.ned-tech.com OR http://ned-tech.com, but where do I make this change?

Thanks again guys! You are awesome!

**Comment by RatexIndex** [comment]
On all html files
<a href="https://steamcommunity.com/openid/login?openid.ns=http%3A%2F%2Fspecs.openid.net%2Fauth%2F2.0&openid.mode=checkid_setup&openid.return_to=http%3A%2F%2Fwww.csgowinbig.com%2Fphp%2Fsteam-login-handle.php&openid.realm=http%3A%2F%2Fwww.csgowinbig.com&openid.ns.sreg=http%3A%2F%2Fopenid.net%2Fextensions%2Fsreg%2F1.1&openid.claimed_id=http%3A%2F%2Fspecs.openid.net%2Fauth%2F2.0%2Fidentifier_select&openid.identity=http%3A%2F%2Fspecs.openid.net%2Fauth%2F2.0%2Fidentifier_select" class="btn btn-lg btn-success" id="login-main" style="display: none; margin-bottom: 10px;">

and in CSGOWinBig/src/php/SteamAuthentication/steamauth/settings.php
$steamauth['domainname'] = "http://www.csgowinbig.com/"; // The main URL of your website displayed in the login page
